rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/data/JobInfoImpl.java

```diff
@@ -294,6 +297,15 @@ public void setVariables(Map<String, String> variables) {
         this.variables = variables;
     }
 
+    @Override
+    public Map<String, JobVariable> getVariablesFull() {
+        return variablesFull;
+    }
+
```

Comment:
add @Override ?

Reply:
this setter is not defined in the interface